x-pack/plugins/enterprise_search/public/applications/shared/react_router_helpers/format_route.ts

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
I wonder if we could remove this library altogether and use [generatePath](https://reactrouter.com/core/api/generatePath) function from react-router. It seems to be doing the same thing.